test/e2e/multicluster/ubernetes_lite.go

```diff
@@ -378,9 +380,8 @@ type StaticPVTestConfig struct {
 
 // Check that the pods using statically created PVs get scheduled to the same zone that the PV is in.
 func PodsUseStaticPVsOrFail(f *framework.Framework, podCount int, image string) {
-	// TODO: add GKE after enabling admission plugin in GKE
```

Comment:
Link PR that "enables admission plugin"?

Reply:
It's not an open source PR :-)